client/src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import './style/index.css';
```

Comment:
remove this

Reply:
Remove index.css and move it into App.css, just next to App.js component - and only import it there